src/Nethermind/Nethermind.Consensus.AuRa/AuRaContractGasLimitCalculator.cs

```diff
@@ -21,13 +21,12 @@
 using Nethermind.Core;
 using Nethermind.Core.Caching;
 using Nethermind.Core.Crypto;
-using Nethermind.Crypto;
 using Nethermind.Dirichlet.Numerics;
 using Nethermind.Logging;
 
 namespace Nethermind.Consensus.AuRa
 {
-    public class AuRaContractGasLimitOverride : IGasLimitOverride
+    public class AuRaContractGasLimitCalculator : IGasLimitCalculator
```

Comment:
can we call it AuRaContractGasLimitOverrideCalculator ? ContractGasLimitOverride is kind og domain name

Reply:
done, thanks